Makefile

```diff
@@ -6,7 +6,13 @@ RS_AST_OUT = rust/ast/src/types_generated.rs \
 	rust/ast/src/visit_generated.rs \
 	rust/ast/src/source_location_accessor_generated.rs \
 	rust/generated_parser/src/stack_value_generated.rs
-PYTHON = python3
+PYTHON = $(shell pwd)/jsparagus_build_venv/bin/python3
+
+init:
+	python3 -m venv jsparagus_build_venv &&\
+	source jsparagus_build_venv/bin/activate &&\
+	pip install --upgrade pip &&\
```

Comment:
Instead of `pip`, use `$(PIP)`.

Reply:
(This matters because, as I understand it, $VENV_BIN/pip installs packages locally inside the venv directory, but if you get `/usr/local/bin/pip`, it'll install the packages globally, which is always bad.)